prac_05/wimbledon.py

```diff
@@ -0,0 +1,43 @@
+def read_wimbledon_data(filename):
+    champions = {}  # Dictionary to store champions and their win counts
```

Comment:
Dictionary naming structure, champions implies a list here, champions_to_?

Reply:
I agree looking back just having champions doesn't fit the naming conventions and is a bit ambiguous as well as it also stores the wins not just the names. 
Would something such as champions_to_wins work in this case?

Reply:
In the display champions function of your code your for loop iterates the items of the dictionary as champion, wins so champions_to_wins makes perfect sense. The emails program has the same thing with user_info dictionary. The iteration in the for loop is perfect naming for your dictionaries just your dictionary is named differently haha

Reply:
`champion` (singular) is better than `champions` (plural), which still implies a list and so would be misleading.